cvat-core/src/annotations-collection.js

```diff
@@ -810,11 +810,13 @@
                         'The object has not been saved yet. Call annotations.put([state]) before',
                     );
                 }
-
-                const distance = object.constructor.distance(state.points, x, y);
-                if (distance !== null && (minimumDistance === null || distance < minimumDistance)) {
-                    minimumDistance = distance;
-                    minimumState = state;
+                if (!(object instanceof Tag)) {
```

Comment:
I would suggest to modify a condition above instead:

``
if (state.outside || state.hidden || state.objectType === ObjectType.TRACK) continue;
``